freight/utils/auth.py

```diff
@@ -1,18 +1,30 @@
 from __future__ import absolute_import
 
-from flask import request, session
+from flask import current_app, request, session
 
 from freight.models import User
 
+from freight.testutils.fixtures import Fixtures
+
 NOT_SET = object()
 
 
 def get_current_user():
     """
     Return the currently authenticated user based on their active session.
+    Will return a dummy user if in development mode.
     """
     if getattr(request, 'current_user', NOT_SET) is NOT_SET:
-        if session.get('uid') is None:
+        if current_app.config.get('DEV'):
+            request.current_user = User.query.filter(
+                User.name == 'Freight',
+            ).first()
+            if not request.current_user:
+                request.current_user = Fixtures().create_user(
```

Comment:
This feels a little crappy but it's the only place where the Freight user is defined. LMK if it's worth pulling it out of fixtures and giving it it's own util.